cgmes/cgmes-conformity/src/main/resources/conformity-modified/cas-1.1.3-data-4.0.3/MicroGrid/BaseCase/BC_BE_v2_line_disconnected_at_boundary_node/MicroGridTestConfiguration_BC_BE_SSH_V2.xml

```diff
@@ -0,0 +1,286 @@
+<?xml version="1.0" encoding="utf-8"?>
+<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:cim="http://iec.ch/TC57/2013/CIM-schema-cim16#" xmlns:entsoe="http://entsoe.eu/CIM/SchemaExtension/3/1#" xmlns:md="http://iec.ch/TC57/61970-552/ModelDescription/1#">
+  <md:FullModel rdf:about="urn:uuid:52b712d1-f3b0-4a59-9191-79f2fb1e4c4e">
```

Comment:
We have an issue with CGMES conversion tests. The CI spends 20 minutes to run them. It seems that they are too big in general and more comparable to integration tests than to unit tests. We have to think about this issue with @flo-dup and maybe think about smallest network as we have the export now.

Reply:
> We have an issue with CGMES conversion tests. The CI spends 20 minutes to run them. It seems that they are too big in general and more comparable to integration tests than to unit tests. We have to think about this issue with @flo-dup and maybe think about smallest network as we have the export now.

We will think about it, there are multiple alternatives: we either could "strip down" the test cases, removing the non-essential data, run all tests only on some builds/under a specific configuration, or move some of them to a separate repo.